cast/tests/declare_test/Scarb.toml

```diff
@@ -0,0 +1,12 @@
+[package]
```

Comment:
Let's move this to conform to: https://github.com/software-mansion/protostar/pull/2140

Reply:
btw, this is planned structure for declare tests: https://github.com/software-mansion/protostar/commit/166c3191d242a2e48d72bb370078fd668180f90e

Reply:
Ok I will remove the tests.

Reply:
[remove tests](https://github.com/software-mansion/protostar/pull/2138/commits/38e8c6d78bf31316cb103b03da1d19389cf70db3)